### PR TITLE
add disclaimer about 3rd party packages

### PR DESCRIPTION
**Pull request by jtgrassie**
Demoted the 3rd party maintained package installs to after compiling from repository source and removed the broken Homebrew package instructions.

### CODE REVIEW

**Comment by jtgrassie** [comment]
Welcome critique of disclaimer copy and any Homebrew users whoe could test https://github.com/monero-project/monero/issues/3786#issuecomment-387927166
If that works I can update this PR to add the Homebrew package instructions back in.

**Comment by jtgrassie** [comment]
Added new Homebrew package instructions (credit @sammy007).

**Comment by jtgrassie** [comment]
It's moved specially so it's lower down in the file. Demoted. It's preferable to build from the project sources only, not to use a third party package that is maintained externally. Yes it's annoying for the diff, but I believe it's warranted in this case.

File: README.md [hunk 1]
```diff
@@ -489,6 +446,46 @@ By default, in either dynamically or statically linked builds, binaries target t
 * ```make release-static-win64``` builds binaries on 64-bit Windows portable across 64-bit Windows systems
 * ```make release-static-win32``` builds binaries on 64-bit or 32-bit Windows portable across 32-bit Windows systems
 
+## Installing Monero from a package
+
+**DISCLAIMER: These packages are not part of this repository or maintained by this projects contributers, and as such, do not go through the same review process to ensure their trustworthiness and security.**
```
Comment (moneromooo-monero) [inline README.md]: projects's contributors

Comment (jtgrassie) [reply README.md]: +1

**Comment by ilovezfs** [comment]
This formula is in Homebrew/homebrew-core. All that's needed is

```
brew install monero
```
to use the prebuilt bottle binaries.

There is no reason to use third-party taps or source builds.

**Comment by sammy007** [comment]
@ilovezfs your formula is always outdated or broken.

**Comment by ilovezfs** [comment]
@sammy007 FYI we have nearly the highest rate of fresh packages amongst all package managers. See https://repology.org/

I suggest you adjust your expectations given the reason it was outdated was that you opened a PR that did not pass CI and I needed to complete the pull request for you.

**Comment by ilovezfs** [comment]
Maybe try to operate from facts before you insult people: https://github.com/Homebrew/homebrew-core/graphs/contributors

**Comment by sammy007** [comment]
Your understanding of "insult" seems very special. What a point of referring to contributors? If you can't update formula in advance before mandatory protocol upgrades just remove it, many people will thank you.

@jtgrassie maybe it's not stupid to just remove all references 3rd party packages ffs.

**Comment by ilovezfs** [comment]
>Maybe somebody else can explain to you how it's necessary to keep shit up to date. I am out.

>What a point of referring to contributors?

My point is that I've done more work to keep Homebrew/homebrew-core up to date than anyone else, I'm the lead maintainer of Homebrew/homebrew-core, and I don't need you to explain the importance of keeping things up to date to me.

**Comment by sammy007** [comment]
> My point is that I've done more work to keep Homebrew/homebrew-core

Great, thank you. But the formula which is a subject of this issue was outdated for a while that's why README suggests tap. Would be awesome to have just one formula in brew core, but it's not always up to date when it needed the most and could lead to money loss.

**Comment by ilovezfs** [comment]
>was outdated

The key word is "was" and I'm sorry that was the case. If you have a problem with the current formula or prebuilt bottles, I'm all ears but I don't see any reason people should be using a forked tap not the core formula at this point, nor any reason Homebrew should be delisted.

**Comment by ilovezfs** [comment]
>Hint: major.minor.rev, when major or minor changed, it must be upgraded ASAP in advance before HF. This is what is special about it.

OK.

**Pull request closed without merging**